File: C/Program06.c
/////////////////////////////////////////////////////////////////////////
//                                                                     //
//  File name :     Program6.c                                         //
//  Description :   Used to print sum of N natural numbers             //
//  Author :        Pallavi Omprakash Malewar                          //
//  Date :          28/04/2025                                         //
//                                                                     //
/////////////////////////////////////////////////////////////////////////


#include<stdio.h>

// Iteration 

int Calculate(int iNumber)
{
    int iCount = 0 , iSum = 0 ;

    for(iCount = 1 ; iCount <= iNumber ; iCount++)
    {
        iSum = iSum + iCount;
    }
    
    return iSum;
}


int main()
{
    int iValue = 0 , iRet = 0 ;

    printf("Enter the Value : ");
    
    scanf("%d",&iValue);

    iRet = Calculate(iValue);

    printf("Summation is : %d\n" , iRet);

    return 0;
}

File: JAVA/Program06.java
///////////////////////////////////////////////////////////////////////////
//                                                                       //
//  File name :     Program06.java                                       //
//  Description :   Used to Display Sum of Factors of Given Number       //
//  Author :        Pallavi Omprakash Malewar                            //
//  Date :          24/08/2025                                           //
//                                                                       //
///////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Number
{
    public int sumOfFactors(int iNo)
    {
        int iCount = 0 , iSum = 0;

        for(iCount = 1 ; iCount <= (iNo / 2) ; iCount++)
        {
            if((iNo % iCount) == 0)
            {
                iSum += iCount;
            }
        }
        iSum += iNo;        // To add that Number itself

        return iSum;
    }
}

class Program06
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0, iRet = 0;

        System.out.print("Enter Number : ");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        iRet = nobj.sumOfFactors(iValue);

        System.out.println("Sum of Factors of "  + iValue + " is " + iRet);

        sobj.close();
    }
}